client/lib/signup/themes-data.js

```diff
@@ -45,12 +45,12 @@ export const themes = [
 		verticals: []
 	},
 	{
-		name: 'Dyad',
-		slug: 'dyad',
+		name: 'Dyad 2',
```

Comment:
Ahh, yes. Just catching up with the thread you started on the themes p2. I’ll make that adjustment now.